modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/acquisitionannotation/v0.1.0.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpactsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/acquisitionannotation/v0.1.0/shapes/"
```

Comment:
"acquisitionannotation" is used here as schema name while "ImageAcquisitionAnnotation" is the activity type in the [morphology-reconstruction-prov-template](https://github.com/BlueBrain/nexus-bbp-domains/blob/master/docs/src/main/paradox/assets/provtemplates/morphology-reconstruction-prov-template.svg). Shouldn't we have "imageacquisitionannotation" as schema name ?

Reply:
I will the update the prov pattern, makes more sense.